website/docs/components/tag/partials/guidelines/guidelines.md

```diff
@@ -44,3 +44,15 @@ When the dismiss icon is trailing, the mouse user must adjust positioning of the
 ## Content
 
 When content within the tag is not user-generated, it should be concise and consistent with the category name.
+
+### Truncation
+
+The text container of the tag has a max-width of 150px and will truncate text if any content overflows beyond that limit. If truncation is present, a tooltip will appear with the full text, on hover or focus of the text container. By default, the tooltip's placement will be top, however, this can be customized if necessary.
```

Comment:
[suggetion] smaller sentences and more active voice

```suggestion
The text container of the tag has a max-width of 150px. The text will truncate if it goes beyond that limit. A tooltip will display the full text when the user hovers or focuses on the truncated text. The tooltip will be placed at the top by default, but this can be customized.
```